gpflow/models/gpmc.py

```diff
@@ -62,7 +63,13 @@ def __init__(
             loc=to_default_float(0.0), scale=to_default_float(1.0)
         )
 
-    def log_likelihood(self, *args, **kwargs) -> tf.Tensor:
+    def log_posterior_density(self) -> tf.Tensor:
+        return self.log_likelihood() + self.log_prior_density()
+
+    def maximum_log_likelihood_objective(self) -> tf.Tensor:
+        return self.log_likelihood()
+
+    def log_likelihood(self) -> tf.Tensor:
```

Comment:
in `GPR` below, you've used `log_marginal_likelihood`, but here the `log_likelihood`. why's that? if the technical explanation is complicated, maybe we can have it offline, separately from the PR